src/main/kotlin/com/cognifide/gradle/aem/common/pkg/PackageOptions.kt

```diff
@@ -9,7 +9,7 @@ class PackageOptions(private val aem: AemExtension) : Serializable {
     /**
      * Package specific configuration
      */
-    val configDir = aem.obj.projectDir(aem.prop.string("package.configPath") ?: "src/aem/package")
+    val commonDir = aem.obj.projectDir(aem.prop.string("package.commonPath") ?: "src/aem/package")
```

Comment:
unfortunately - breaking change

now config means OSGi configs stored under /apps/{project}/config
previously config was used in case of common resources for built packages.

we need to distinguish it